docs/blog/posts/bazzite-buzz-07.md

```diff
@@ -0,0 +1,144 @@
+---
+date: 2023-11-07
+comments: true
+authors: 
+  - nicknamenamenick
+categories:
+  - bazzite
+links:
+  - Bazzite Commits: https://github.com/ublue-os/bazzite/commits/main
+  - Bazzite 2.0: https://universal-blue.org/blog/2023/11/07/bazzite-20/
+---
+
+# Bazzite Buzz #7: Fedora 39 Builds Out Now
+
+![bg](https://github.com/ublue-os/website/assets/121328689/830c2afb-1796-43ef-a093-90449503f81c)
+
+<hr>
+
+<p style="text-align: center; font-weight: 600; color: red">ATTENTION:</p>
+
+If you tested the Fedora 39 builds of Bazzite early, then you need to rebase back to `:latest` because `:39` will be unsupported eventually.
```

Comment:
"... because 39 will be unsupported eventually" doesn't sit right with me because it sounds like we're going to be ending support sometime soon to someone that may be unfamiliar with Fedora's release schedule. I would say something along the lines of "... to be updated to future major Bazzite versions"